front/src/common/api/generatedEditoastApi.ts

```diff
@@ -888,7 +869,10 @@ const injectedRtkApi = api
       >({
         query: (queryArg) => ({
           url: `/v2/timetable/${queryArg.id}/conflicts`,
-          params: { infra_id: queryArg.infraId },
+          params: {
+            infra_id: queryArg.infraId,
+            electrical_profile_set_id: queryArg.electricalProfileSetId,
+          },
```

Comment:
When creating a scenario without electrical profile, I have this error : 

![Capture d’écran 2024-07-30 à 17 55 11](https://github.com/user-attachments/assets/4b90d3a0-a566-4682-970b-e85c693ba4df)

Reply:
Oh no

Reply:
Ok my bad, I didn't rebuild editoast after your last changes